app/modules/color_utils.c

```diff
@@ -1,12 +1,12 @@
 #include "module.h"
 #include "lauxlib.h"
 #include "lmem.h"
-#include "platform.h"
+//#include "platform.h"
```

Comment:
AFAICS it is OK to remove these includes. Or do I need to `#ifndef LUA_CROSS_COMPILER` them?

Reply:
If it compiles without the include then you can remove it. :wink: TBH, this is the quickest way to find out.  I removed a load of redundant includes in the modules doing this trial and error method. However, in this case, I would expect it to compile fine.

Reply:
Done